pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
+    return x / y
```

Comment:
Hello, I think there is a mistake here. I think this should be written:
```suggestion
    return x / z
```
Cheers, James